packages/react-examples/src/react/Nav/Nav.Wrapped.Example.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { Nav, INavStyles, INavLinkGroup } from '@fluentui/react/lib/Nav';
+
+const navStyles: Partial<INavStyles> = {
+  root: {
+    width: 208,
+    height: 350,
+    boxSizing: 'border-box',
+    border: '1px solid #eee',
+    overflowY: 'auto',
+  },
+  link: {
+    whiteSpace: 'normal',
+    lineHeight: 'inherit',
+  },
+};
+
+const navLinkGroups: INavLinkGroup[] = [
+  {
+    links: [
+      {
+        name: 'Home',
+        url: 'http://example.com',
+        expandAriaLabel: 'Expand Home section',
+        collapseAriaLabel: 'Collapse Home section',
+        title: '',
+        links: [
+          {
+            name: 'Activity',
+            url: 'http://msn.com',
+            key: 'key1',
+            target: '_blank',
+            title: '',
+          },
+          {
+            name: 'MSN',
+            url: 'http://msn.com',
+            disabled: true,
+            key: 'key2',
+            target: '_blank',
+            title: '',
+          },
+        ],
+        isExpanded: true,
+      },
+      {
+        name: 'Shared Documents and Files',
+        url: 'http://example.com',
+        key: 'key3',
+        isExpanded: true,
```

Comment:
Not super important but I don't think this prop makes sense for a link without children?